interpreter/tests/interpreter.rs

```diff
@@ -448,6 +450,20 @@ icmp_test!(exec_icmp_uge, "uge", [(0, 0), (1, 0)]);
 //     }
 // }
 
+#[cfg(all(target_arch = "aarch64", target_vendor = "apple"))]
+#[test]
+fn exec_printf() {
```

Comment:
Did you consider not to execute `cargo` from this test? We would rather not execute external commands.